### PR TITLE
feat(runtime): support conda install to use configs.channels field in the runtime.yaml

### PR DESCRIPTION
**Pull request by tianweidut**
## Description
- #741 

## Modules
- [x] Client

## Checklist
- [x] run code format and lint check
- [ ] add unit test
- [ ] add necessary doc

### CODE REVIEW

File: client/starwhale/utils/venv.py [hunk 1]
```diff
@@ -707,6 +722,7 @@ def _do_restore_venv(
     _python_version: str,
     _wheels: t.Optional[t.List[str]],
     _deps: t.Optional[t.Dict[str, t.Union[t.List[str], str]]],
+    _configs: t.Optional[t.Dict[str, t.Dict[str, t.Union[str, t.List[str]]]]],
```
Comment (jialeicui) [inline client/starwhale/utils/venv.py:725]: consider type alias? 
https://docs.python.org/3/library/typing.html#type-aliases

**Comment by jialeicui** [comment]
LGTM

File: client/starwhale/utils/venv.py (hunk 1 above)
Comment (tianweidut) [reply client/starwhale/utils/venv.py:725]: ok

Comment (tianweidut) [reply client/starwhale/utils/venv.py]: updated

**Review by jialeicui — APPROVED** [review]

**Pull request merged**